flaml/automl.py

```diff
@@ -665,6 +665,7 @@ def custom_metric(
                 provided in custom_hp. Note that the value for "domain" can either be a constant
                 or a sample.Domain object.
                 e.g.,
+            skip_transform: boolean, default=False | Whether to pre-process data prior to modeling
```

Comment:
Please move this line after 681. Line 669-081 is part of the example for `custom_hp`.
Also, please add a period in the end.

Reply:
whoops, sorry about that